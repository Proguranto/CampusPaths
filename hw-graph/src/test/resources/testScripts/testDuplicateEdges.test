# Test what happens when adding two of the same edges
# to a simple graph with 1 node.

# Create a graph
CreateGraph graph1

# Add a node
AddNode graph1 n1

# Add two of the same edges
AddEdge graph1 n1 n1 e1
AddEdge graph1 n1 n1 e1

# Print the nodes in the graph and the outgoing edges from n1
ListNodes graph1
ListChildren graph1 n1