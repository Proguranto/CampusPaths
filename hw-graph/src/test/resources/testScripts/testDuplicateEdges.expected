# Test what happens when adding two of the same edges
# to a simple graph with 1 node.

# Create a graph
created graph graph1

# Add a node
added node n1 to graph1

# Add two of the same edges
added edge e1 from n1 to n1 in graph1
added edge e1 from n1 to n1 in graph1

# Print the nodes in the graph and the outgoing edges from n1
graph1 contains: n1
the children of n1 in graph1 are: n1(e1)